pages/index.js

```diff
@@ -1,10 +1,14 @@
 import Head from "next/head";
+// import Header from './components/Header'
 
 import styled from "styled-components";
 
 import HackSC from "../assets/hacksc.svg";
 import WhiteFlower from "../assets/white_flower.svg";
 
+const Ava = () => {
```

Comment:
lol nice

Reply:
lol i was trying to test how to make a component and nothing was showing up and then i realized i needed to `npm run dev`